docs: documentation for new contract burndown feature (#2439)
langchain-ai/docs: src/langsmith/billing.mdx

1. Invite members to your organization, as desired.
1. Enter your credit card information. Then, enter business information, invoice email, and tax ID. If this organization belongs to a business, check the **This is a business** checkbox and enter the information accordingly. For more information, refer to the [Update your information section](#update-your-information).

## Track contract usage (Enterprise)

<Note>
Contract burndown tracking is available for **Enterprise plan** customers with prepaid commitments. You must have the `organization:manage` permission to access this feature.

> If you are interested in the Enterprise plan, please [contact sales](https://www.langchain.com/contact-sales).

Reviewer: ```suggestion
For more details on the Enterprise plan, [contact the sales team](https://www.langchain.com/contact-sales).
```

Author: updated
